packages/components/src/font-size-picker/utils.ts

```diff
@@ -142,13 +145,27 @@ export function getToggleGroupOptions(
 
 export function getSelectedOption(
 	fontSizes: FontSize[],
-	value: FontSizePickerProps[ 'value' ]
+	value: FontSizePickerProps[ 'value' ],
+	useSelectControl: boolean,
+	disableCustomFontSizes: boolean = false
 ): FontSizeOption {
```

Comment:
The return type here is `FontSizeOption` but your changes make it so that the return type is `FontSizeSelectOption | FontSizeToggleGroupOption`. This should be a type error but I guess it's not because of the `// @ts-ignore`.

Reply:
Oh good point. That was an oversight. The `// @ts-ignore` is for something else entirely. I'll update.

Reply:
On the other PR coz I just merged 😄

Reply:
Here's what I need to do to make TS shut up

```patch
diff --git a/packages/components/src/font-size-picker/index.tsx b/packages/components/src/font-size-picker/index.tsx
index 3a1f792c04..66c666be05 100644
--- a/packages/components/src/font-size-picker/index.tsx
+++ b/packages/components/src/font-size-picker/index.tsx
@@ -118,7 +118,7 @@ const UnforwardedFontSizePicker = (
 		disableCustomFontSizes
 	);
 
-	const isCustomValue = selectedOption.slug === CUSTOM_FONT_SIZE;
+	const isCustomValue = selectedOption?.slug === CUSTOM_FONT_SIZE;
 	const [ showCustomValueControl, setShowCustomValueControl ] = useState(
 		! disableCustomFontSizes && isCustomValue
 	);
@@ -226,7 +226,7 @@ const UnforwardedFontSizePicker = (
 									options as FontSizeSelectOption[]
 								 ).find(
 									( option ) =>
-										option.key === selectedOption.slug
+										option.key === selectedOption?.slug
 								) }
 								onChange={ ( {
 									selectedItem,
diff --git a/packages/components/src/font-size-picker/types.ts b/packages/components/src/font-size-picker/types.ts
index ec193f51bf..1df66bb922 100644
--- a/packages/components/src/font-size-picker/types.ts
+++ b/packages/components/src/font-size-picker/types.ts
@@ -92,6 +92,7 @@ export type FontSizeSelectOption = Pick< FontSizeOption, 'size' > & {
 
 export type FontSizeToggleGroupOption = {
 	key: string;
+	slug: string;
 	value: number | string;
 	label: string;
 	name: string;
diff --git a/packages/components/src/font-size-picker/utils.ts b/packages/components/src/font-size-picker/utils.ts
index 02427e0419..b59c3609a2 100644
--- a/packages/components/src/font-size-picker/utils.ts
+++ b/packages/components/src/font-size-picker/utils.ts
@@ -135,6 +135,7 @@ export function getToggleGroupOptions(
 	return optionsArray.map( ( { slug, size, name }, index ) => {
 		return {
 			key: slug,
+			slug,
 			value: size,
 			size,
 			label: labelAliases[ index ],
@@ -148,7 +149,7 @@ export function getSelectedOption(
 	value: FontSizePickerProps[ 'value' ],
 	useSelectControl: boolean,
 	disableCustomFontSizes: boolean = false
-): FontSizeOption {
+): FontSizeOption | FontSizeToggleGroupOption {
 	if ( ! value ) {
 		return DEFAULT_FONT_SIZE_OPTION;
 	}
```